.github/workflows/build.yml

```diff
@@ -28,6 +28,10 @@ jobs:
     runs-on: ubuntu-latest
     env:
       BUILDKIT_INLINE_CACHE: 1
+      CACHE_IMAGE: "ghcr.io/ibm/text-gen-router:build-cache"
```

Comment:
I would remove `DOCKER_BUILDKIT: 1` and `BUILDKIT_INLINE_CACHE: 1`

Even though I hope it would not interfere with the `setup-buildx-action` and the cache modes of the `build-push-action`, it should be obsolete.

Reply:
It looks like `CI=true` is always set by github as well